src/data-table/column-categorical.js

```diff
@@ -56,16 +58,17 @@ function InputBefore() {
 }
 
 function FilterQuickControls(props: {
+  locale: LocaleT,
```

Comment:
Can this component `useContext` rather than shuttle the locale through props? No need to include the type if that's possible

Reply:
Sure. 👍